fabtests/multinode/src/core_coll.c

```diff
@@ -525,7 +525,8 @@ static inline int setup_hints()
 	hints->mode = FI_CONTEXT;
 	hints->domain_attr->control_progress = FI_PROGRESS_MANUAL;
 	hints->domain_attr->data_progress = FI_PROGRESS_MANUAL;
-	hints->fabric_attr->prov_name = strdup("tcp");
+	if (!hints->fabric_attr->prov_name)
```

Comment:
I wonder if we can just remove this completely and either let the `-p` option set it or let getinfo decide

Reply:
We should remove this and align the behavior with all other tests.